.github/ISSUE_TEMPLATE/bug_report.yaml

```
@@ -0,0 +1,64 @@
+name: Bug Report
+description: Report a problem in PTSD
+labels: [bug]
+body:
+
+  - type: markdown
+    attributes:
+      value: |
+        *Before reporting:*
+        - Confirm the problem is reproducible on **master** or **latest stable** release
+        - Search [existing issues](https://github.com/ntut-open-source-club/practical-tools-for-simple-design/issues?q=is%3Aissue+label%3Abug)
+        - Include `cmake --system-information` for build-related issues.
+  - type: textarea
+    attributes:
+      label: "Problem"
+      description: "Describe the current behavior. May include logs, images, or videos."
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: "Steps to reproduce"
+      description: |
+        - If the bug pertains to crashing (or segfault), please include a stacktrace.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: "Expected behavior"
+      description: "Describe the behavior you expect."
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: "PTSD version"
+      placeholder: "013246df758a14ea083bfbe7726a589db19a560a"
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: "Operating system/version"
+      placeholder: "macOS 11.5"
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: "CMake version"
+      placeholder: "3.27.8"
+    validations:
+      required: true
+
```

More toolchain info would be nice.
Check [the msg here](https://discord.com/channels/1093814266630512681/1093938857772322886/1206121976351952926).

---

test if i can put suggestion here
```suggestion
  - type: textarea
    attributes:
      label: "CMakeCache.txt"
      description: |
        Can found in the build dir:
        - Clion (`cmake-build-debug/CMakeCache.txt`)
        - Visual Studio (`out\build\x64-Debug\CMakeCache.txt`)
        - Others (`build/CMakeCache.txt`)
        please use `Attach File` do not copy paste whole file.
    validations:
      required: true

```

---

@NOOBDY please take a look